packages/zoe/src/contractSupport/atomicTransfer.js

```diff
@@ -1,8 +1,13 @@
 import { mustMatch, M } from '@agoric/store';
 import { assertRightsConserved } from '../contractFacet/rightsConservation.js';
+import { AmountKeywordRecordShape, SeatShape } from '../typeGuards.js';
 
 const { Fail, quote: q } = assert;
 
+export const TransferPartShape = M.arrayOf(
+  M.or(SeatShape, AmountKeywordRecordShape),
+);
```

Comment:
Seems surprisingly loose. Trying a tighter pattern at https://github.com/Agoric/agoric-sdk/pull/7040 , but do not yet know what CI has to say about it.

Have not yet looked at the `TransferPart` type, but clearly they should be co-maintained